src/doc.c

```diff
@@ -2325,6 +2327,40 @@ void highlightText(Scope *sc, Dsymbol *s, OutBuffer *buf, size_t offset)
                 }
                 break;
 
+            case '`':
+                if (inBacktick)
+                {   inBacktick = 0;
```

Comment:
Don't use this style of bracing please (regardless if some of it existed already), thanks. :)